services-geojson/src/main/java/com/mapbox/geojson/GeometryCollection.java

```diff
@@ -99,6 +100,20 @@ public static GeometryCollection fromGeometries(@NonNull List<Geometry> geometri
     return new AutoValue_GeometryCollection(null, geometries);
   }
 
+  /**
+   * Create a new instance of this class by giving the collection a single GeoJSON {@link Geometry}.
+   *
+   * @param geometry a non-null object of type geometry which makes up this collection
+   * @return a new instance of this class defined by the values passed inside this static factory
+   *   method
+   * @since 3.0.0
+   */
+  public static GeometryCollection fromGeometry(@NonNull Geometry geometry) {
+    List<Geometry> geometries = new ArrayList<>();
+    geometries.add(geometry);
+    return new AutoValue_GeometryCollection(null, geometries);
```

Comment:
Since there is not much logic to these three lines, you can alternatively just do a one liner instead: `return new AutoValue_GeometryCollection(null, new ArrayList<>(){geometry});`

Reply:
tried making this change but `new ArrayList<>(){geometry}` doesn't work. The closest thing I got to work was `new ArrayList<>(1){Geometry geometry;}` but in that case, I think I'm just creating a new geometry object? Have a better example of this working properly? I can fix it in a future PR. Merging this one now.

Reply:
ah makes sense, I'll make sure to fix that up in the next sweep